web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -581,6 +601,16 @@ class TestRunView extends Component {
                 }
               })()}
             </Collapsible>
+            <Collapsible trigger="WSO2 Server logs" lazyRender={true} triggerWhenOpen="WSO2 Server logs >>" >
```

Comment:
It seems the collapsible top bar is a bit too tall. Shell reduce the height a bit? This is there in all the collapsibles actually.